src/main/java/uk/gov/hmcts/divorce/document/CaseDataDocumentService.java

```diff
@@ -77,7 +74,7 @@ public Document renderDocument(final Map<String, Object> templateContent,
         return documentFrom(documentInfo);
     }
 
-    private void updateCaseData(final CaseData caseData,
+    public void updateCaseData(final CaseData caseData,
```

Comment:
why is this public now?

Reply:
Trying to keep in line with what you were saying about making an update to case data an explicit call. Before it was a step when rendering the document exclusively